ui/src/app/services/conversion/conversion.service.ts

```diff
@@ -331,6 +331,7 @@ export class ConversionService {
           Name: '',
           GenerationType: ''
         },
+        srcDefaultValue: data.SrcSchema[tableId].ColDefs[colId].DefaultValue.Value.Query,
```

Comment:
Will this not throw an error if data.SrcSchema[tableId].ColDefs[colId].DefaultValue is null or undefined?